dlp/src/test/java/com/example/dlp/DeIdentificationIT.java

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.dlp;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.util.regex.Pattern;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+@RunWith(JUnit4.class)
+public class DeIdentificationIT {
+  private ByteArrayOutputStream bout;
+  private PrintStream out;
+
+  // Update to wrapped local encryption key
+  private String wrappedKey = System.getenv("DLP_DEID_WRAPPED_KEY");
```

Comment:
We'll need to update our testing if this stays.

Reply:
I've added these environment variables to Travis, but they (and a few BigQuery tables) are based off of the `nodejs-docs-samples` project - and will need to be moved over at some point.